src/chunk.js

```diff
@@ -56,29 +75,6 @@ export function startupChunk(nodeOrAmpDoc, fn) {
 };
 
 /**
- * Run the given function sometime in the future without blocking UI.
- *
- * Higher priority tasks are executed before lower priority tasks.
- * Tasks with the same priority are executed in FIFO order.
- *
- * Uses `requestIdleCallback` if available and passes the `IdleDeadline`
- * object to the function, which can be used to perform a variable amount
- * of work depending on the remaining amount of idle time.
- *
- * @param {!Node|!./service/ampdoc-impl.AmpDoc} nodeOrAmpDoc
- * @param {function(?IdleDeadline)} fn
- * @param {ChunkPriority} priority
- */
-export function chunk(nodeOrAmpDoc, fn, priority) {
-  if (deactivated) {
-    resolved.then(fn);
-    return;
-  }
-  const service = fromClassForDoc(nodeOrAmpDoc, 'chunk', Chunks);
```

Comment:
Agreed, that's actually what I'm doing in this PR.

- Extensions use `chunksForDoc` -> `getExistingServiceForDoc`
- Entry points continue to use `startupChunk` -> `fromClassForDoc` to install the service as a side effect

Did you mean I should change `startupChunk` to use `getExistingServiceForDoc` and explicitly install the service in each entry point?